NuGet.config

```
@@ -5,7 +5,8 @@
     <!-- ensure only the sources defined below are used -->
     <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
     <add key="xamarin" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/xamarin-impl/nuget/v3/index.json" />
```

Ok, LOL the new source:

https://aka.ms/maui-preview/index.json

Just redirects to:

https://pkgs.dev.azure.com/azure-public/vside/_packaging/xamarin-impl/nuget/v3/index.json

The .NET MAUI team must have made this, I was not aware of it.

If you want to just change the above line:

```suggestion
    <add key="maui" value="https://aka.ms/maui-preview/index.json" />
```

I think we should probably use all lower case & short words for the keys, thanks!